### DISCUSSION TITLE
external access?

### DISCUSSION THREAD

**Discussion by RodGama** [nathaneltitane/dextop#10] [Q&A] [answered]
How do i acquire external access? For now i can only use in localhost, my pc can't seem to find the vnc server

**Comment by nathaneltitane** [comment, marked as answer]
> External access?

you need to modify 'container-vnc' and remove the 'localhost' directive...
I will be reviewing the utility to be more flexible and default to 'localhost' if no address is specified.

**Reply by nathaneltitane** [reply]
Dextop's 'container-vnc' funcionality has now been merged into 'container-session':
Address resolution is only set to 'localhost' (127.0.0.1) when ${address} is empty or not specified.
